packages/calypso-products/src/plans-list.js

```diff
@@ -11,6 +10,11 @@ import i18n, { translate } from 'i18n-calypso';
 import { isEnabled } from '@automattic/calypso-config';
 import * as constants from './constants';
 
+/* eslint-disable wpcalypso/jsx-classname-namespace */
```

Comment:
Can we move this below where it's necessary, and perhaps re-enable it after that line? Or use `eslint-disable-line` instead

Reply:
Good point. Not sure why that ended up there.